.github/workflows/upload.yml

```diff
@@ -22,14 +22,6 @@ jobs:
         with:
           go-version: '1.21'
           check-latest: true
-      - name: Cache Plugins
-        id: cache-plugins
-        uses: actions/cache@v3
-        with:
-          path: downloads
-          key: ${{ runner.os }}-plugin-downloads-${{ hashFiles('**/plugin-releases.json') }}
-          restore-keys: |
-            ${{ runner.os }}-plugin-downloads
```

Comment:
The cache got too large to be of any value - we're downloading the files from GitHub releases and so these should run quickly from the GH runners.